src/app.ts

```diff
@@ -250,18 +253,45 @@ export function newApp({
   //
   // Setup parsers
   //
+  if (!DISABLE_BODY_PARSER) {
+    // Parse the incoming request body. This is needed by Passport spid strategy.
+    app.use(bodyParser.json());
 
-  // Parse the incoming request body. This is needed by Passport spid strategy.
-  app.use(bodyParser.json());
-
-  // Parse an urlencoded body.
-  app.use(bodyParser.urlencoded({ extended: true }));
+    // Parse an urlencoded body.
+    app.use(bodyParser.urlencoded({ extended: true }));
+  }
 
   //
   // Define the folder that contains the public assets.
   //
 
-  app.use(express.static("public"));
+  app.get(/(\.html|\.svg|\.png)$/, async (req, res, next) => {
+    try {
+      const path = "public" + req.path;
+      if (!fs.existsSync(path)) {
+        return next();
+      }
+      const stat = await fs.promises.stat(path);
+      if (stat.isDirectory()) {
+        return next();
+      }
+      const type = mime.lookup(path);
+      if (type) {
+        const charset = mime.charsets.lookup(type);
+        res.setHeader(
+          "Content-Type",
+          // tslint:disable-next-line: restrict-plus-operands
+          type + (charset ? `; charset=${charset}` : "")
+        );
+      }
+      const content = await fs.promises.readFile(path);
+      res.setHeader("Content-Length", stat.size);
+      res.status(200).send(content);
+    } catch (err) {
+      log.error(`static|Error retrieving static file asset|error:%s`, err);
+      return next(err);
+    }
```

Comment:
It's not clear to me why we introduced this behaviour. Besides that, we can put this in a middleware function `(req, res, next)=>void` and we can give this chunk an explicit meaning.

Reply:
This logic is required because the express default `static` folder ([ref](https://expressjs.com/it/starter/static-files.html)) doesn't work inside Azure functions. Here we emulate what `static` middleware does. We can create a specific middleware, but this code will be used only on this endpoint.

Reply:
we can just move it in a function declared inside the same module

Reply:
Fixed in 94282ce